### ISSUE TITLE
Allow ignoring embeds from Discord

### ISSUE THREAD

**Issue by fortepc** [Bjornskjald/miscord#198]
_If you have a question, ask on the [Miscord Support server](https://discord.gg/DkmTvVz) instead_  

## Suggested Behavior
Config option to allow the bridge to not pass over embeds. 

## Reason for Request
Some bots have very rich embeds that when translated to Facebook Messenger turns into a giant wall of text. Being able to tell Miscord to just throw away the entire embed and only grab non-embed messages may be desirable for some.

I don't feel like this is a super important feature, and I'm not sure if I'd use it, but some people have complained about how the embed turns into a giant wall of text. 

## Examples

![image](https://user-images.githubusercontent.com/26149601/38752151-391671c0-3f28-11e8-8dad-fce4c946a95e.png)

![image](https://user-images.githubusercontent.com/26149601/38752270-92f32fc6-3f28-11e8-94ec-1afe7ce95f40.png)

![image](https://user-images.githubusercontent.com/26149601/38752173-4b1420a2-3f28-11e8-95d6-864319f33cbd.png)

Additionally, if you could give a third option and convert the embed into an image to attach, that'd be even better.

**Comment by Bjornskjald** [comment]
> convert the embed into an image to attach
That would mean adding a rendering engine just to render an embed and add it to the message

**Comment by fortepc** [comment]
Yeah, the convert embed into an image would be just a wishlist kind of thing, but the option to totally ignore embeds would still be a good feature to bake in without that option.
